### DISCUSSION TITLE
Seeking Feedback: Integrating Zedd into Zed IDE

### DISCUSSION THREAD

**Discussion by TulipJani** [zed-industries/zed#14675] [Feature Requests]
Hey Zed Fam,

I've created Zedd [check here](https://github.com/TulipJani/Zedd/) ~an AI companion. I'm planning to integrate for Zed IDE, and I'd love your quick thoughts on how to integrate it best. Drop your feedback below—and let me know what do you think

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
